src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const createNotification = (message, isSuccess) => {
+  const notification = document.createElement('div');
+
+  notification.className = `notification ${isSuccess ? 'success' : 'error'}`;
+  notification.dataset.qa = 'notification';
+  notification.textContent = message;
+  document.body.appendChild(notification);
+};
+
+const firstPromise = new Promise((resolve, reject) => {
+  const handleLeftClick = (e) => {
+    resolve('First promise was resolved');
+    document.removeEventListener('click', handleLeftClick);
+  };
+
+  document.addEventListener('click', handleLeftClick);
+
+  setTimeout(() => {
+    reject(new Error());
```

Comment:
The error object in the rejection of the first promise does not provide a meaningful message. Consider providing a descriptive error message to help with debugging or user feedback.